### ISSUE TITLE
Do you have any idea why torchvision vgg16 cannot converge with your training pipeline?

### ISSUE THREAD

**Issue by yxchng** [amdegroot/ssd.pytorch#57]
Do you have any idea why torchvision vgg16 cannot converge with your training pipeline? I just changed your vgg model to torchvision ones and it can't converge. I tried many different parameters

**Comment by chauhan-utk** [comment]
Can you post a little more details?

**Comment by yxchng** [comment]
@chauhan-utk Basically i just swap the vgg model with the torchvision version, the rest are the same and I couldn't get it to converge.

**Comment by amdegroot** [comment]
The torchvision VGG16 is different because it is not the fully convolutional reduced (atrous) model.
To quote the SSD arxiv paper, "[the model] is pre-trained on the ILSVRC CLS-LOC dataset. Similar to DeepLab-LargeFOV, we convert fc6 and fc7 to convolutional layers, subsample parameters from fc6 and fc7, change pool5 from 2 × 2 − s2 to 3 × 3 − s1, and use the a trous ` algorithm to fill the
”holes”. We remove all the dropout layers and the fc8 layer."
